maintenance/maintenance_test.go

```diff
@@ -111,6 +111,19 @@ func checkAccounts() (bool, error) {
 }
 
 const deviceList = `---
+brand:                  Generic
+codename:               AndroidTablet270dpi
+form:                   VIRTUAL
+formFactor:             TABLET
+id:                     AndroidTablet270dpi
+manufacturer:           Generic
+name:                   Generic 720x1600 Android tablet @ 270dpi
+screenDensity:          270
+screenX:                720
+screenY:                1600
+supportedAbis[0]:       x86
+supportedVersionIds[0]: 30
+---
```

Comment:
Did you get this device and its properties from somewhere on Firebase? I am just curious because I did not had to work on this step.

Reply:
We have a test for this 😎: https://github.com/bitrise-steplib/steps-virtual-device-testing-for-android/blob/master/maintenance/maintenance_test.go#L17

The test case compares the new device list to the last saved device list, if there is a change, prints the device table, which can be copied to the step.yml.